components/src/core/hero/hero.component.scss

```diff
@@ -1,3 +1,17 @@
+@import '../utility/utility';
+
+/* RTL */
+[dir='rtl'] .pxb-hero {
+    .pxb-hero-primary-wrapper.pxb-hero-svgIcon svg {
+        transform-origin: top right;
```

Comment:
We're not flipping hero svg icons?

Reply:
![image](https://user-images.githubusercontent.com/29152776/88656619-9a475e00-d09e-11ea-88ed-6d2c66e3017d.png)
![image](https://user-images.githubusercontent.com/29152776/88656638-a2070280-d09e-11ea-9d87-b999e852829e.png)

Reply:
These svg icons will require a little more attention than others; the approach of just applying a scaleX(-1) and adjusting the transform origin is not working for these.

I got it this close:
![image](https://user-images.githubusercontent.com/6538289/88659880-c6191280-d0a3-11ea-8f7a-3efc7756d9e2.png)

Reply:
I'm going to capture this requirement as a separate bug since I'm thinking this will take a while to resolve.

Reply:
#123 Added